docs/sources/administration/api-keys/_index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: 'API keys in Grafana'
+menuTitle: 'API keys'
+description: 'This contains information about API keys in Grafana'
+aliases: [docs/sources/administration/api-keys/_index.md]
+weight: 300
+keywords:
+  - Api keys
+  - Service accounts
+---
+
+# API keys in Grafana
+
+An API key is a encrypted string that identifies as a specific role to your Grafana instance when talking to APIs. You can more easily use Grafana HTTP APIs by leveraging API keys.
```

Comment:
```suggestion
An API key is an encrypted string that identifies as a specific role to your Grafana instance when talking to APIs. You can more easily use Grafana HTTP APIs by leveraging API keys.
```

Reply:
I'm a bit confused by "identifies as a specific role" wording here. Could you elaborate please what is your intended message? 

I think simply saying that API Keys can be used to interact with Grafana HTTP APis would be sufficient to be honest.

Reply:
👍